liquidity/liquidity.go

```diff
@@ -220,6 +243,10 @@ func (p Parameters) validate(minConfs int32) error {
 		return ErrZeroMinerFee
 	}
 
+	if p.AutoFeeBudget < 0 {
```

Comment:
Are there some sanity checks we can perform on the start date as well? Not sure if it makes sense or is even necessary...

Reply:
Could add a check that the start date isn't in the future? Other than that, I do like being able to set a 0 value start date, which just means it's a budget for all time?

Reply:
Yeah, I thought the same, maybe it is useful to turn it on later with a future date. Let's leave it the way it is then.